integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -657,6 +662,32 @@ public String getFileSystemName() {
     return mFsName;
   }
 
+  @Override
+  public void umount() {
+    // Release operation is async, we need to make sure
+    // all out stream is closed before umount the fuse
+    LOG.info("Waiting for all file out stream closed");
+    long startTime = System.currentTimeMillis();
+    while (!mCreateFileEntries.isEmpty()) {
+      try {
+        if (System.currentTimeMillis() - startTime >= MAX_UMOUNT_WAITTIME_MS) {
+          throw new TimeoutException("reach the max umount waiting time");
+        }
+        Thread.sleep(100);
```

Comment:
Try not to use Thread.sleep() in the source code, thanks!

Reply:
Fixed, thx